src/api/api.ts

```diff
@@ -52,6 +55,20 @@ export function useCheckSummaries() {
       // Enable retry if the check type has a retry annotation
       checkSummary[Severity.High].checks[checkType].canRetry = canRetry;
       checkSummary[Severity.Low].checks[checkType].canRetry = canRetry;
+      // Get the steps that are ignored for the check type
+      const ignoreSteps = check.metadata.annotations?.[IGNORE_STEPS_ANNOTATION_LIST];
+      if (ignoreSteps) {
+        const steps = ignoreSteps.split(',');
+        for (const step of steps) {
+          delete checkSummary[Severity.High].checks[checkType].steps[step];
+          delete checkSummary[Severity.Low].checks[checkType].steps[step];
+          if (Object.keys(checkSummary[Severity.Low].checks[checkType].steps).length === 0) {
+            // Remove the check type if all steps are ignored
+            delete checkSummary[Severity.High].checks[checkType];
```

Comment:
Likely showing my complete lack of domain knowledge here but could there ever be a case where `checkSummary[Severity.Low].checks[checkType].steps` is empty but `checkSummary[Severity.High].checks[checkType].steps` isn't?